style.css

```diff
@@ -1,7 +1,122 @@
-body div:last-of-type {
-  border: 1px solid blue;
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 0.625rem;
+}
+
+body {
+  background-color: rgba(5, 84, 202, 0.815);
+  padding: 2rem;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.header h1 {
+  font-family: "Montserrat", sans-serif;
+  font-weight: 700;
+  letter-spacing: 1rem;
+  margin-bottom: 2.5rem;
+  color: rgb(88, 6, 6);
+  padding: 0.7rem;
+}
+
+#display-books {
+  padding: 0.7rem;
+  margin-bottom: 3rem;
+}
+
+.single-book {
+  margin-bottom: 2rem;
+  padding: 1rem 1.5em;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border: 0.5px solid white;
+  color: white;
+}
+
+.single-book:nth-child(even) {
+  background-color: rgb(77, 5, 73);
+}
+
+.single-book:nth-child(odd) {
+  background-color: rgb(29, 4, 87);
+}
+
+.book-info {
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.2em;
+}
+
+.book-title,
+.book-author {
+  padding: 0.3em 1em;
+}
+
+.removebtn {
+  background: rgb(40, 4, 65);
+  outline: none;
+  border: 1px solid orangered;
+  padding: 0.5em 1.5em;
+  color: red;
+  font-family: "Montserrat", sans-serif;
+  letter-spacing: 1.6px;
+  font-weight: 200;
+  font-size: 1.2rem;
+  transition: all 0.5s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+  border-radius: 20%;
+}
+
+.removebtn:hover {
+  background: red;
+  border: 1px solid white;
+  color: white;
+  padding: 0.5em 1.5em;
+}
+
+form {
+  padding: 1em;
+}
+
+input[placeholder="Title"],
+input[placeholder="Author"] {
+  border: none;
+  width: 40%;
+  padding: 0.5em;
+  border-radius: 1000px;
+  outline: none;
+  font-size: 1rem;
+}
+
+.add-btn {
+  background: rgb(40, 4, 65);
+  outline: none;
+  border: 1px solid orangered;
+  padding: 0.25em 0.5em;
+  color: red;
+  font-family: "Montserrat", sans-serif;
+  letter-spacing: 1.6px;
+  font-weight: 200;
+  font-size: 1.2rem;
+  transition: all 0.5s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+  border-radius: 20%;
```

Comment:
- [ ] The add button should be positioned to the right and not the center as from the given layout.
- yours
![image](https://user-images.githubusercontent.com/68167810/124914362-edd56200-dff8-11eb-8a60-604565f809dc.png)
- template
![image](https://user-images.githubusercontent.com/68167810/124914567-2f660d00-dff9-11eb-9816-d5769a4bd673.png)